pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "instana"
+dynamic = [
+    "version",
+]
+description = "Python Distributed Tracing & Metrics Sensor for Instana."
+readme = "README.md"
+requires-python = ">=3.7"
+license = "MIT"
+keywords = [
+    "performance",
+    "opentracing",
+    "metrics",
+    "monitoring",
+    "tracing",
+    "distributed-tracing",
+]
+authors = [
```

Comment:
Shouldn't the authors list contain the list of contributors of this repository or the current team members of python tracers?

If we decide to go generic, I would suggest the name to be something like `"Instana Team Python Tracer Engineers"`

Reply:
We could just reference https://github.com/orgs/instana/teams/eng-python

Reply:
The [specification](https://packaging.python.org/en/latest/specifications/pyproject-toml/#authors-maintainers) states that:
> The name value MUST be a valid email name (i.e. whatever can be put as a name, before an email, in [RFC 822](https://datatracker.ietf.org/doc/html/rfc822.html)) and not contain commas.

So, as `Instana Team Python Tracer Engineers` is a valid email name, I modified it to this name.